docs/source/routing/observability/router-telemetry-otel/apm-guides/datadog/connecting-to-datadog/otel-collector.mdx

```diff
@@ -0,0 +1,368 @@
+
+The collector uses a pipeline architecture with four key components:
+- **Receivers**: Collect telemetry from sources (in this case, the router via OTLP)
+- **Processors**: Transform and modify data (batching, sampling, filtering)
+- **Connectors**: Generate new telemetry from existing data (the Datadog connector creates trace metrics)
+- **Exporters**: Send data to backends (Datadog in this configuration)
+
+```mermaid
+flowchart LR
+    Router[Apollo Router] -->|OTLP| OTLP[OTLP Receiver]
+
+    subgraph Collector[OpenTelemetry Collector]
+        %% Traces Pipeline
+        OTLP -->|Traces| BatchT[Batch Processor]
+        BatchT --> Connector[Datadog Connector]
+        Connector --> DatadogExp[Datadog Exporter]
+
+        %% Metrics Pipeline
+        OTLP -->|Metrics| BatchM[Batch Processor]
+        Connector -->|Trace Metrics| BatchM
+        BatchM --> DatadogExp
+    end
+
+    DatadogExp --> Datadog[Datadog APM]
+
+```
+
+For detailed configuration options, see [configuration basics](https://opentelemetry.io/docs/collector/configuration/) and [Datadog's collector exporter documentation](https://docs.datadoghq.com/opentelemetry/setup/collector_exporter/).
+
+
+Create `otel-collector.yaml` with this minimal configuration. Set your Datadog API key as an environment variable: `export DD_API_KEY=your_datadog_api_key_here`:
+
+```yaml title="otel-collector.yaml"
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+
+processors:
+  batch:
+    # Datadog APM intake limit is 3.2MB
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+
+connectors:
+  # Required for APM trace metrics
+  datadog/connector:
+
+exporters:
+  datadog/exporter:
+    api:
+      key: ${env:DD_API_KEY}
+      # site: datadoghq.eu # Uncomment for EU site
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [datadog/connector, datadog/exporter]
+
+    metrics:
+      receivers: [otlp, datadog/connector]
+      processors: [batch]
+      exporters: [datadog/exporter]
+```
+
+## Verify the connection
+
+With both the router and OpenTelemetry collector running, verify that telemetry data is flowing to Datadog:
+
+1. **Send test queries** to your router to generate telemetry data
+
+2. **View your data in Datadog**:
+   - **Service**: Go to [APM > Services](https://app.datadoghq.com/apm/services) and look for the `apollo-router` service
+   - **Metrics**: Go to [Metrics > Explorer](https://app.datadoghq.com/metric/explorer) and search for `http.server.request.duration` filtered by `service:apollo-router`
+
+## Enhance your configuration
+
+After you verify the basic setup, you can enhance your configuration with additional features.
+
+### Router enhancements
+
+Add [router instrumentation](/graphos/routing/observability/router-telemetry-otel/apm-guides/datadog/router-instrumentation) for Datadog-optimized span attributes, error tracking, and better APM organization.
```

Comment:
so the datadog agent trace page has [an explanation](https://www.apollographql.com/docs/deploy-preview/30564f93bcb5bcd157c64fcd/graphos/routing/observability/router-telemetry-otel/apm-guides/datadog/connecting-to-datadog/datadog-agent/datadog-agent-traces#batch_processor) on the batch processor (which should arguably live under `Exporters` because it doesn't just apply to datadog agent tracing). This page should also mention it as well, because batch_processor applies to exporting metrics _and_ tracing for otlp, as well as metrics being sent to apollo. 

We get a lot of questions about batch processing, so I would really love if we have a detailed explanation on how to best tweak all those values, as well as the relationship between those values in router.yaml vs otel-collector.yaml.